### PR TITLE
bump chainlink-relay; update relayer.Health()

### PR DESCRIPTION
**Pull request by jmank88**
`chainSet.Healthy()` is no longer available on latest chainlink-relay

Smoke tests depend on -solana, but fortunately they are not required b/c it depends on this also:
- https://github.com/smartcontractkit/chainlink-solana/pull/484

### CODE REVIEW

**Review by essamhassan — APPROVED** [review]
thanks Jordan!

File: relayer/pkg/chainlink/relay.go [hunk 1]
```diff
@@ -51,9 +51,7 @@ func (r *relayer) Ready() error {
 	return r.chainSet.Ready()
 }
 
-func (r *relayer) Healthy() error {
-	return r.chainSet.Healthy()
-}
+func (r *relayer) Healthy() error { return nil }
```
Comment (essamhassan) [inline relayer/pkg/chainlink/relay.go:54]: Any reason why we still need this and not remove all of it?

Comment (essamhassan) [reply relayer/pkg/chainlink/relay.go:54]: same question for solana

Comment (jmank88) [reply relayer/pkg/chainlink/relay.go:54]: I was just trying to do the minimum to fix things and thought that might create more of the same problem :shrug:

**Pull request merged**